src/main/java/ch/njol/skript/expressions/ExprNow.java

```diff
@@ -15,24 +15,56 @@
 import ch.njol.skript.util.Date;
 import ch.njol.util.Kleenean;
 
+import java.time.DateTimeException;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
 @Name("Now")
 @Description("The current <a href='classes.html#date'>system time</a> of the server. Use <a href='#ExprTime'>time</a> to get the <a href='classes.html#time'>Minecraft time</a> of a world.")
 @Examples({"broadcast \"Current server time: %now%\""})
```

Comment:
needs new example

Reply:
Also would be good to update the description to it's known how to use timezones.